includes/class-paybutton-admin.php

```diff
@@ -146,6 +158,17 @@ public function paywall_settings_page() {
         $this->load_admin_template( 'paywall-settings', $args );
     }
 
+    public function admin_notice_missing_ecash_address() {
+        if (!current_user_can('manage_options')) {
+            return;
+        }
+        $address = get_option('paybutton_paywall_ecash_address', '');
+        if (empty($address)) {
+            echo '<div class="notice notice-error">';
+            echo '<p><strong>PayButton:</strong> Please set your eCash Address in <a href="' . esc_url(admin_url('admin.php?page=paybutton-paywall')) . '">Paywall Settings</a>. If you don\'t have an eCash Address yet, create a wallet using <a href="https://cashtab.com">Cashtab</a> or <a href="https://www.bitcoinabc.org/electrum/">Electrum ABC</a>.</p>';
```

Comment:
Same here.

We can link to Cashtab and Electrum ABC for XEC, and Electron Cash for BCH. Wording would need some updating to make it clear that we support both.

I think in the future if we do expand to other blockchains, we'll have to improve the text layout a bit to make it more manageable as we don't want the banner at the top taking up so much room; although maybe it's fine since we want them to address the "error" anyway.